crates/typst-library/src/meta/counter.rs

```diff
@@ -388,6 +388,29 @@ impl Counter {
         UpdateElem::new(self.0, update).pack()
     }
 
+    /// Calculate the difference between the real page number and the logical one.
+    ///
+    /// Difference between last counter change.
```

Comment:
the signature makes very unclear what it takes and what it returns (logical, real). also, is it useful to return the _difference_ here or would we rather get just logical number from real number?